app/views/account/AccountSettingsView.coffee

```diff
@@ -83,6 +84,12 @@ module.exports = class AccountSettingsView extends CocoView
       confirmModal.on 'confirm', @resetProgress
       @openModalView confirmModal
 
+  onClickResendVerificationEmail: (e) ->
+    $.post "/db/user/#{me.id}/request-verify-email", ->
+      link = $(e.currentTarget)
+      link.find('.resend-text').addClass('hide')
+      link.find('.sent-text').removeClass('hide')
```

Comment:
We decided to [map endpoints to model and collection functions](https://app.asana.com/0/search/136119551918263/68016868411685), so this definitely should. We didn't explicitly decide on endpoints that do not map to a db collection (like /contact), but I think it makes sense, by extension, to have module functions of the same argument structure, so the way views interact with the server is uniform.